src/test/java/com/puppycrawl/tools/checkstyle/MainTest.java

```diff
@@ -78,41 +77,43 @@
           + " configuration XML file%n"
           + "that configures the checks to apply.%n"
           + "      <files>...            One or more source files to verify%n"
-          + "      --executeIgnoredModules%n"
-          + "                            Allows ignored modules to be run.%n"
-          + "      --tabWidth=<tabWidth> Sets the length of the tab character. Used only with"
-          + " \"-s\" option. Default%n"
-          + "                              value is 8%n"
+
           + "  -c=<configurationFile>    Sets the check configuration file to use.%n"
           + "  -C, --checker-threads-number=<checkerThreadsNumber>%n"
           + "                            (experimental) The number of Checker threads (must be"
           + " greater than zero)%n"
           + "  -d, --debug               Print all debug logging of CheckStyle utility%n"
           + "  -e, --exclude=<exclude>   Directory/File path to exclude from CheckStyle%n"
+          + "      --executeIgnoredModules%n"
+          + "                            Allows ignored modules to be run.%n"
           + "  -f=<format>               Sets the output format. Valid values: xml, plain."
           + " Defaults to plain%n"
           + "  -g, --generate-xpath-suppression%n"
           + "                            Generates to output a suppression.xml to use to suppress"
-          + " all violations from%n"
-          + "                              user's config%n"
+          + " all violations%n"
+          + "                              from user's config%n"
           + "  -h, --help                Show this help message and exit.%n"
           + "  -j, --javadocTree         Print Parse tree of the Javadoc comment%n"
           + "  -J, --treeWithJavadoc     Print full Abstract Syntax Tree of the file%n"
           + "  -o=<outputPath>           Sets the output file. Defaults to stdout%n"
           + "  -p=<propertiesFile>       Loads the properties file%n"
           + "  -s=<suppressionLineColumnNumber>%n"
           + "                            Print xpath suppressions at the file's line and column"
-          + " position. Argument is%n"
-          + "                              the line and column number (separated by a : ) in the"
-          + " file that the%n"
-          + "                              suppression should be generated for%n"
+          + " position.%n"
+          + "                              Argument is the line and column number (separated by a"
+          + " : ) in the%n"
+          + "                              file that the suppression should be generated for%n"
           + "  -t, --tree                Print Abstract Syntax Tree(AST) of the file%n"
           + "  -T, --treeWithComments    Print Abstract Syntax Tree(AST) of the file including"
           + " comments%n"
+          + "      --tabWidth=<tabWidth> Sets the length of the tab character. Used only with"
```

Comment:
This looks like a bad regression.

tabWidth is not part of '-T'

Reply:
@remkop could confirm, I assumed it was sorting by options, including fully worded ones. That was my first thought but upon closer inspection, `tabWidth` should still come before `treeWithComments`.